parameters/experiments/m13/pursuit_with_noisy_gaze.params

```diff
@@ -5,6 +5,9 @@ _includes:
 experiment: 'pursuit-ablating-gaze'
 curriculum: pursuit
 learner: pursuit-gaze
+accuracy_to_txt : True
+accuracy_logging_path : "out_out.txt"
```

Comment:
Is this supposed to be a full path or just a file name? It would probably be clearer as the accuracy logging file name and programmatically placing it in the directory output where the `index.html` gets placed for the experiment. That way all the outputs from an experiment can be found in the same location by default.

Reply:
Ah sorry this was for debugging and I forgot to take it out -- but yes, I agree :)

Reply:
If its just debug that's fine too! imo, an example param doc showing how to include that file logging probably isn't a bad thing. Can you add it to the `debug.param` ?

Reply:
Yes, will do!